src/backend/api/meals.js

```diff
@@ -23,54 +48,50 @@ router.get("/", async (request, response) => {
       }
     });
 
-    let sortedMeals = knex("meals");
+    let sortedMeals = knex("meals")
+      .select(
+        "idmeals",
+        "title",
+        "max_number_of_guests",
+        "price",
+        "createdAt",
+        "description",
+        "imageURL"
```

Comment:
Seems like this is the same list you use in `router.get("/:id"` - consider extracting this list to a const and then you could do `.select(...selectFields)`.